libpod/image/utils.go

```diff
@@ -40,3 +47,63 @@ func findImageInRepotags(search imageParts, images []*storage.Image) (*storage.I
 	}
 	return results[0], nil
 }
+
+// GetCopyOptions constructs a new containers/image/copy.Options{} struct from the given parameters
+func GetCopyOptions(reportWriter io.Writer, signaturePolicyPath string, srcDockerRegistry, destDockerRegistry *DockerRegistryOptions, signing SigningOptions, authFile, manifestType string, forceCompress bool) *cp.Options {
```

Comment:
Can this be private?

Reply:
We should also look into cleaning this up later, maybe a builder pattern?

Reply:
private